docs/api.md

```diff
@@ -0,0 +1,72 @@
+# Vivify's API
+
+In addition to the command line interface `viv`, Vivify has HTTP endpoints that
+can be used to integrate it into other applications, such as making editor
+plugins for live previews.
+
+## Path encoding
+
+Vivify's endpoints are often accessed with the pattern `/<endpoint>/<path>`
```

Comment:
Would it be fair to say they are always accessed with that pattern? I would consider just dropping the word often.

Reply:
We do have `GET /health` and `DELETE /viewer` at the root of the endpoints with no paths.

We could instead say something like

> **Current wording**
>
> > Vivify's endpoints are often accessed with the pattern `/<endpoint>/<path>` where path is a [percent-encoded](https://developer.mozilla.org/en-US/docs/Glossary/Percent-encoding) path in the file system.
>
> **Suggestion**
>
> > When accessing an endpoint corresponding to a path in the file system, use `/<endpoint>/<path>` where `<path>` is [percent-encoded](https://developer.mozilla.org/en-US/docs/Glossary/Percent-encoding).

Is that better? ^^